kura/distrib/pom.xml

```diff
@@ -32,7 +29,7 @@
     <name>distrib</name>
     <url>http://maven.apache.org</url>
     <properties>
-        <tycho-version>1.6.0</tycho-version>
+        <tycho-version>3.0.5</tycho-version>
```

Comment:
Please remove the java8 profile down in this pom. `skip.js.nashorn.engine.components` should not be needed anymore